### PR TITLE
Adds 'Secret' field to GitHub trigger.

### PR DESCRIPTION
**Pull request by ttomsu**
This is the UI portion of https://github.com/spinnaker/echo/pull/131, and should not be submitted until that PR goes in.

tag: https://github.com/spinnaker/fiat/issues/134

@anotherchrisberry @danielpeach PTAL

### CODE REVIEW

File: app/scripts/modules/core/help/helpContents.js [hunk 1]
```diff
@@ -315,6 +315,7 @@ module.exports = angular.module('spinnaker.core.help.contents', [])
     'pipeline.config.script.propertyFile': '<p>(Optional) The name to the properties file produced by the script execution to be used by later stages of the Spinnaker pipeline. </p>',
     'pipeline.config.docker.trigger.tag': '<p>(Optional) If specified, only the tags that match this Java Regular Expression will be triggered. Leave empty to trigger builds on any tag pushed.</p>',
     'pipeline.config.git.trigger.branch': '<p>(Optional) If specified, only pushes to the branches that match this Java Regular Expression will be triggered. Leave empty to trigger builds for every branch.</p>',
+    'pipeline.config.git.trigger.githubSecret': '<p>(Optional, but recommended) If specified, verifies GitHub as the sender of this trigger. See <a href="https://developer.github.com/webhooks/securing/">GitHub docs</a> for more information.</p>',
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/help/helpContents.js]: should probably add `target="_blank"` to the link - we typically do that on external links so they'll open in a new tab.

Comment (ttomsu) [inline app/scripts/modules/core/help/helpContents.js]: Funny thing - I went to test it out, and once I move my mouse to click on the link, the help text disappears.

Comment (anotherchrisberry) [inline app/scripts/modules/core/help/helpContents.js]: Oh, good point. You can't actually hover on tooltips at all.

We had a similar issue with the alerts/notices. The fix was not straightforward, but could probably be applied to help fields. I'll see what I can do there.

Comment (ttomsu) [inline app/scripts/modules/core/help/helpContents.js]: Added `target="_blank"`

**Pull request merged**